Shared/messages/MessagesOverlay.cpp

```diff
@@ -53,33 +48,31 @@ namespace Dsa {
 /*!
   \brief Constructor taking a \a geoView and an optional \a parent.
  */
-MessagesOverlay::MessagesOverlay(GeoView* geoView, QObject* parent) :
-  MessagesOverlay(geoView, nullptr, QString(), SurfacePlacement::DrapedBillboarded, parent)
-{
-}
 
 /*!
   \brief Constructor taking a \a geoView, a \a renderer, a \a messageType,
   a \a surfacePlacement mode and an optional \a parent.
  */
-MessagesOverlay::MessagesOverlay(GeoView* geoView, Renderer* renderer, const QString& messageType,
-                                 SurfacePlacement surfacePlacement, QObject* parent) :
+
+/*!
+  \brief Destructor.
+ */
+MessagesOverlay::MessagesOverlay(Esri::ArcGISRuntime::GeoView *geoView, Esri::ArcGISRuntime::Renderer *renderer, const QString &messageType, MessageFeed *messageFeed, Esri::ArcGISRuntime::SurfacePlacement surfacePlacement, QObject *parent) :
```

Comment:
You likely don't need the fully qualifies namespaces here.

Also, Qt Creator has opinions about where the "*" and "&" go that don't jive with the rest of the project.

Reply:
Same as above.